starter-code/styles/base.css

```diff
@@ -1,11 +1,13 @@
-/* TODO: Add a rule for images, so they never exceed their container's width. */
-
 body {
   line-height: 1.5;
   font-size: 100%;
   color: #313131;
 }
 
+img {
+  width: 100%;
```

Comment:
`width: 100%` sets the image to the width of the container, we want a rule that says it won't **exceed** the width of the container. You should look up `max-width: 100%` and see what the difference is.